#!/usr/bin/env python
PACKAGE = "pal_vision_segmentation"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("window_height", int_t, 0, "Window height", 10, 1, 640)
gen.add("window_width", int_t, 0, "Window width", 10, 1, 480)
gen.add("coeffs_to_cancel", int_t, 0, "Coefficients to cancel in DCT transform", 2, 0, 10)

exit(gen.generate(PACKAGE, "template_segmentation_node", "TemplateSegment"))
